docs/tutorials/typescript/work-queue-slack-app/index.mdx

```diff
@@ -0,0 +1,991 @@
+You should already have it running if you completed the [Prerequisite steps](#prerequisites).
+So, during development it will seem like you're managing two things instead of one.
+
+However, after the development process, if you use Temporal Cloud as your Temporal Service, you only need to deploy the Temporal Application the same way as you would your Slack App.
+Then, your Temporal Application replaces an entire database with less than 70 lines of code.
+
+![Temporal development vs production](./images/temporal-slack-app-dev-vs-prod.svg)
+
+To use Temporal Cloud in production, see the [Deploy your TypeScript application to Digital Ocean](/tutorials/typescript/digital-ocean-deployment) tutorial for more information.
+
+### Functional requirements
+
+**What are the functional requirements of the application?**
+
+1. Using a Slash command, a user can submit a work request.
+2. Using a Slash command, a user can see all the work requests for the channel.
+3. When viewing all the work requests for the channel, a user can click a button to claim a work request.
+4. When viewing all the work requests for the channel, a user can click a button to mark a work request as complete.
+5. Using a Slash command, a user can delete the Work Queue for the channel.
+
+![App functionality demo](./images/work-queue-demo.gif)
+
+## Create a new Slack App configuration
+
+Go to https://api.slack.com/apps?new_app=1 and create an app "From scratch." Name the app whatever you want, and select a workspace that you intend to install it to.
+
+Once you've done this, click "Create App."
+
+### Enable Socket Mode
+
+Under Settings in the navigation bar, select Socket Mode and toggle ON socket mode for this application.
+
+![Enable Socket Mode](./images/enable-sockets.png)
+
+When you enable Socket Mode, Slack gives you an application token that you will need when you start the app.
+This token will typically start with `xapp`.
+It is located under App-Level Tokens in Basic Information.
+
+### Create a Slash Command
+
+Select "Slash Commands" in the sidebar navigation, and click "Create New Command".
+Add a new command for `/workqueue`.
+Since you have Socket Mode enabled, you won't need to add a request URL.
+
+![Create a Slash Command](./images/enable-slash-command.png)
+
+### Subscribe to Events
+
+Under Features in the navigation bar, select Event Subscriptions and ensure Enable Events is turned ON.
+The navigate to the section under "Subscribe to bot events" and add subscriptions for:
+
+- `message.groups`
+- `message.im`
+- `message.channels`
+
+![Subscribe to bot events](./images/subscribe-to-bot-events.png)
+
+Once you've done this, click "Save Changes"
+
+### Set OAuth & Permissions
+
+Under Features in the navigation bar, go to OAuth & Permissions.
+
+You'll need to enable the following Bot Token Scopes:
+
+- `channels:history`
+- `channels:read`
+- `chat:write`
+- `groups:history`
+- `groups:read`
+- `im:history`
+- `mpim:history`
+
+### Install App
```

Comment:
I think the screenshots do a good job of breaking up these sections and I would turn many/most of these h3s into just bold text, so it doesn't pollute the sidebar too much.

Reply:
See my comment above, after thinking about this - I think removing anchors because of the feeling of TOC pollution is not a tradeoff I would make.